src/EmojiPanel.cc

```diff
@@ -55,7 +53,7 @@ EmojiPanel::EmojiPanel(QWidget *parent)
         contentLayout->setMargin(0);
 
         auto emojiCategories = new QFrame(mainWidget);
-        emojiCategories->setStyleSheet("background-color: #f2f2f2");
+        // emojiCategories->setStyleSheet("background-color: #f2f2f2");
```

Comment:
The emoji categories would need a different background color from the rest of the panel.